supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,19 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[basic-http-authentication]]
+==== image:images/yes.png[yes] Basic HTTP authentication (noun)
+*Description*: _Basic HTTP authentication_, or _Basic authentication_, is an authorization scheme that uses HTTP to send login credentials as user ID/password pairs that are encoded in Base64. 
```

Comment:
Should we try to rephrase this text to avoid the slash?

Reply:
Agree. I'm just going to solve the problem by cutting that bit out as follows `to send login credentials that`...